source/lib-intro.tex

```diff
@@ -1579,13 +1579,8 @@
 \indextext{implementation!freestanding|(}%
 
 \pnum
-Two kinds of implementations are defined:
-\indextext{implementation!hosted}%
-hosted and freestanding\iref{intro.compliance};
-the kind of the implementation is
-\impldef{whether the implementation is hosted or freestanding}.
-For a hosted implementation, this document
-describes the set of available headers.
+For a hosted implementation,
+this document describes the set of available headers.
```

Comment:
It seems a bit odd to say this in the subclause titled "Freestanding implementations".

Separately from that, I think xref to [intro.compliance] for the definitions of host impl and freestanding impl would help.